src/foam/parsers/HTMLLexer.js

```diff
@@ -168,10 +132,9 @@ foam.CLASS({
                    sym('value')))),
 
           value: str(alt(
-              plus(notChars('\'" \t\r\n<>')),
-              // TODO(markdittmer): Support proper escaping inside strings.
-              seq1(1, '"', repeat(notChars('"', anyChar())), '"'),
-              seq1(1, "'", repeat(notChars("'", anyChar())), "'"))),
+              plus(alt(sym('escape'), notChars('\'" \t\r\n<>'))),
```

Comment:
I was required to alter this case as well, since Chrome parses `<option value=&amp;Potato />`